core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -75,7 +75,8 @@ case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName,
                                        roles: Set[ProjectRoles.ProjectRole],
                                        status: CreationStatuses.CreationStatus,
                                        message: Option[String],
-                                       managedAppCoordinates: Option[AzureManagedAppCoordinates]
+                                       managedAppCoordinates: Option[AzureManagedAppCoordinates], // TODO: do we even want to return this?
```

Comment:
The ticket requested that we change the API to provide the cloud platform of a billing project in a clearer way than looking for the presence of `managedAppCoordinates`. I am wondering if we even want to return `managedAppCoordinates` at all, as we don't pass it to the server when using the billing project.

Reply:
Yeah if it's not needed for UI consumption I am +1 on removing it.

Reply:
This is the only place it's used in terra-ui, and it would be good to replace it with the more explicit cloudContext variable. We will have to temporarily make terra-ui handle either one though.

```
const isAzureBillingProject = project => {
    if (project === undefined) {
      project = _.find({ projectName: namespace }, billingProjects)
    }
    // Azure billing projects have `managedAppCoordinates` defined.
    return !!project?.managedAppCoordinates
  }